homeassistant/components/bme280/sensor.py

```diff
@@ -144,6 +143,15 @@ def __init__(self, bme280_client, sensor_type, temp_unit, name):
         self.type = sensor_type
         self._state = None
         self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
+        super().__init__(
+            DataUpdateCoordinator(
```

Comment:
Please don't instantiate the coordinator here. Do that separately before instantiating the entity.